docs/sdk/getting-started/react-native.md

```diff
@@ -146,10 +146,10 @@ Do this integration, if you link React modules using relative path inside your P
     * Add these lines to import section
 
     ```objective-c
-    #import <AppCenterReactNative/AppCenterReactNative.h>
-    #import <AppCenterReactNativeAnalytics/AppCenterReactNativeAnalytics.h>
-    #import <AppCenterReactNativeCrashes/AppCenterReactNativeCrashes.h>
-    #import <AppCenterReactNativePush/AppCenterReactNativePush.h>
+    #import <AppCenterReactNative.h>
```

Comment:
I'd like to understand why changing this.

Is the `<AppCenterReactNative/AppCenterReactNative.h>` not working for you?

Reply:
Yes, it throws `'AppCenterReactNative/AppCenterReactNative.h' file not found`
Same for all the other dependencies (`<AppCenterReactNativeAnalytics/AppCenterReactNativeAnalytics.h>`, `<AppCenterReactNativeCrashes/AppCenterReactNativeCrashes.h>` and `<AppCenterReactNativePush/AppCenterReactNativePush.h>`)
https://ibb.co/LgCWmzS